saleor/webhook/transport/shipping.py

```diff
@@ -116,6 +116,7 @@ def get_excluded_shipping_methods_or_fetch(
             event_type,
             payload,
             webhook,
+            False,
```

Comment:
This is something that should be passed from previous layer. This function is implicitly called by WebhookPlugin, and WebhookPlugin should set it based on `self.allow_replica`